src/bulkExport/getCrawlerStatus.test.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+import * as AWSMock from 'aws-sdk-mock';
+import AWS from 'aws-sdk';
+import each from 'jest-each';
+import { BulkExportStateMachineGlobalParameters } from './types';
+import { getCrawlerStatusHandler } from './getCrawlerStatus';
+
+AWSMock.setSDKInstance(AWS);
+
+describe('getCrawlerStatusHandler', () => {
+    beforeEach(() => {
+        process.env.CRAWLER_NAME = 'crawlerName';
+        AWSMock.restore();
+    });
+
+    each([
```

Comment:
We should add the two error states:
- Unknown last crawl status: ${lastCrawlStatus}
- Unknown crawler state: ${crawlerState}

Reply:
I did not test those errors on purpose.  

Those 2 code branches exist but they are unreachable unless Glue makes a breaking API change, which is extremely unlikely to happen. A test case like "when Glue does a breaking change on their response structure, then getCrawlerStatus should fail with <error msg>" isn't really useful.

How does that sound to you?

https://docs.aws.amazon.com/glue/latest/webapi/API_LastCrawlInfo.html#Glue-Type-LastCrawlInfo-Status
https://docs.aws.amazon.com/glue/latest/webapi/API_Crawler.html#Glue-Type-Crawler-State

Reply:
As a general rule of thumb, I generally add unit tests to cover all of the code path for the "system under test". In this case the "system under test" has a code path for throwing these "unknown" errors.

However, that is a good point that these edge cases are unlikely to be encountered during run time. 

Personally I would just cover the edge case since it's easy to do so, and as a general rule as stated above. However, this is not a blocking change for me.

Reply:
Good to know context and the implausibility of the edge cases. I'd prefer to add them but not blocking for me either